For_Andrey/main1.1.py

```diff
@@ -0,0 +1,163 @@
+﻿import random  # from random import randint (вызов - randint(2,5))
+import math
+import operator  # сортировка
+
+
+class Car:
+    """Use as template for creating cars"""

```

Comment:
Bad class docstring.
Dosctring should contain correct info about the purpose class is used for